src/js/components/modals/AppModalComponent.jsx

```diff
@@ -131,7 +137,7 @@ var AppModalComponent = React.createClass({
       let app = AppFormStore.app;
 
       if (this.props.app != null) {
-        AppsActions.applySettingsOnApp(app.id, app, true);
+        AppsActions.applySettingsOnApp(app.id, app, true, this.state.force);
       } else {
         AppsActions.createApp(app);
```

Comment:
I was wondering if we could have the same kind of error when "createApp" is called.